### DISCUSSION TITLE
i am a novice . connect mouse to raspberry pi pico to computer

### DISCUSSION THREAD

**Discussion by AbhishekAKA47** [adafruit/Adafruit_TinyUSB_Arduino#278] [Q&A]
https://github.com/adafruit/Adafruit_TinyUSB_Arduino/tree/master/examples/DualRole/HID/hid_device_report
this is the code i want to use to connect the raspberry pico to the mouse and computer but i dont know which wire goes were.
i have connected a usbafemale to the raspberry pi pico using the pins vbus to vbus gnd to gnd d+ to gpio20 d- to gpio 21 but dont know what to do now . i want to receive the hid report from the mouse on the computer as if it was a mouse connected directly to the computer

**Comment by hathach** [comment]
pin is changed to 16, 17. You should recieve report (output to Serial) from mouse when plugging your mouse
